subsys/bluetooth/mesh/Kconfig

```diff
@@ -301,6 +301,26 @@ config BT_MESH_ADV_EXT
 
 endchoice
 
+if BT_MESH_ADV_EXT
+
+config BT_MESH_RELAY_ADV_SETS
+	int "Maximum of simultaneous relay message support"
+	default 0
+	range 0 BT_EXT_ADV_MAX_ADV_SET
+	depends on BT_MESH_RELAY
+	help
+	  Maximum of simultaneous relay message support. Requires controller support
+	  multiple advertising sets.
+
+config BT_MESH_GATT_ADV
+	bool "Use a separate extended advertising set for GATT Server Advertising"
```

Comment:
May be rename it to `BT_MESH_ADV_EXT_GATT_SEPARATE` or so? The current name doesn't describe the purpose of this option.